process/incubation.md

```diff
@@ -158,7 +159,7 @@ Changes to the Swift Server Ecosystem index page will be announced by the SSWG u
 ## Graduation Requirements
 
 * [Minimal Requirements](#minimal-requirements)
-* Have stable API (no pending/planned breaking API changes), with at least one published major release (e.g. 1.0.0)
+* Have stable API (no pending/planned breaking API changes before submission), with at least one published major release (e.g. 1.0.0). Breaking changes are allowed following semantic versioning rules, but major versions should not be released regularly to aid adoption.
```

Comment:
submission is not a word used in the doc; before graduation i guess?

Reply:
Graduation or proposal is probably best